_includes/scripts.html

```diff
@@ -138,4 +138,32 @@
             $("span.cell-content").contents().unwrap();
         }
     }
-</script>
\ No newline at end of file
+</script>
+<script type="text/javascript">
+    function openModal() {
+      document.getElementById("screenshot1").style.display = "block";
+    }
+    
+    function closeModal() {
+      document.getElementById("screenshot1").style.display = "none";
+    }
+    
+    function currentSlide(n) {
+      showSlides(slideIndex = n);
+    }
+    
+    function showSlides(n) {
+      var slides = document.getElementsByClassName("screenshot1Lightbox");
+      slides[slideIndex-1].style.display = "block";
+    }
+    
+    window.onkeyup = function (event) {
```

Comment:
Please add comments to so that anyone reading the code understands what/why this is being done

Reply:
Added.